spec/controllers/application_controller_spec.rb

```diff
@@ -2,18 +2,29 @@
 
 describe ApplicationController do
   it "returns a 200 when correct basic auth credentials used" do
+    env = Rails.env
+    Rails.env = 'production'
```

Comment:
if this throws an error where the code is run, this could end up effecting env for cascading tests (as we probably test in random order).

You might do something like:

```
env = Rails.env
begin
  Rails.env = 'production'

  http_login(ENV['SLUG'], ENV['PASSWORD'])
  # the rest of your test
ensure
  Rails.env = env
end
```

Reply:
updated at 92449e3